src/options.h

```diff
@@ -172,7 +174,8 @@ class options_manager : public JsonSerializer, public JsonDeserializer
         void add( const std::string sNameIn, const std::string sPageIn,
                   const std::string sMenuTextIn, const std::string sTooltipIn,
                   const int iMinIn, int iMaxIn, int iDefaultIn,
-                  copt_hide_t opt_hide = COPT_NO_HIDE );
+                  copt_hide_t opt_hide = COPT_NO_HIDE,
+                  const std::string &format = "%i" );
```

Comment:
`options_manager` clearly needs refactoring: 9th parameter was added to two of six monstrous overloaded functions.